internal/commands/config_pull_policy.go

```diff
@@ -0,0 +1,81 @@
+package commands
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	pubcfg "github.com/buildpacks/pack/config"
+
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/internal/style"
+	"github.com/buildpacks/pack/logging"
+)
+
+func ConfigPullPolicy(logger logging.Logger, cfg config.Config, cfgPath string) *cobra.Command {
+	var unset bool
+
+	cmd := &cobra.Command{
+		Use:     "pull-policy <if-not-present | always | never>",
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "List, set and unset the global pull policy used by other commands",
+		Aliases: []string{"pull-policy"},
+		Long: bpRegistryExplanation + "\n\nYou can use this command to list, set, and unset the default pull policy that will be used when working with containers:\n" +
+			"* To list your pull policy, run `pack config pull-policy`.\n" +
+			"* To set your pull policy, run `pack config pull-policy <pull-policy>`.\n" +
+			"* To unset your pull policy, run `pack config pull-policy --unset`.\n" +
+			"Unsetting the pull policy will reset the policy to the default, which is always",
+		RunE: logError(logger, func(cmd *cobra.Command, args []string) error {
+			switch {
+			case unset:
+				if len(args) > 0 {
+					return errors.Errorf("pull policy and --unset cannot be specified simultaneously")
+				}
+				oldPullPolicy := cfg.PullPolicy
+				cfg.PullPolicy = ""
+				if err := config.Write(cfg, cfgPath); err != nil {
+					return errors.Wrapf(err, "writing config to %s", cfgPath)
+				}
+
+				pullPolicy, err := pubcfg.ParsePullPolicy(cfg.PullPolicy)
+				if err != nil {
```

Comment:
Don't necessarily need to check for error here, as the pull policy has been set to one (line 34 this file) that we know is valid. But there could easily be more succinct ways to do this.